javatests/google/registry/export/ExportConstantsTest.java

```diff
@@ -82,12 +82,15 @@ public void testReportingKinds_areSubsetOfBackupKinds() throws Exception {
     assertThat(ExportConstants.getBackupKinds()).containsAllIn(ExportConstants.getReportingKinds());
   }
 
-  @Test
-  public void testReportingEntityClasses_areAllBaseEntityClasses() throws Exception {
-    for (Class<? extends ImmutableObject> clazz : ExportConstants.REPORTING_ENTITY_CLASSES) {
-      assertThat(clazz.isAnnotationPresent(Entity.class))
-          .named(String.format("class %s is an @Entity", clazz.getSimpleName()))
-          .isTrue();
-    }
-  }
+  @Test public void testReportingKinds_count() throws Exception { 
+    assertThat(ExportConstants.getReportingKinds().size()).isEqualTo(28); 
+  }  
+
+  @Test public void testReportingKinds_areSubsetOfEntityClasses() throws Exception {  
+    ImmutableSet<String> entityClasses =  FluentIterable.from(EntityClasses.ALL_CLASSES) 
+                                          .filter(hasAnnotation(Entity.class)) 
```

Comment:
Incorrect formatting for line continuation.  It should be indented by +4 spaces, not a lot.